packages/api/tests/cli/test-back-restore.js

```diff
@@ -61,27 +61,10 @@ test.serial('backup records from DynamoDB', async (t) => {
   const stats = fs.statSync(tempBackupFile);
   t.truthy(stats);
 
-  // delete records
-  await gModel.batchWrite(granuleIds.map((id) => ({ granuleId: id })));
-});
-
-test.serial('restore records to DynamoDB', async (t) => {
-  const limit = 25;
-  const granuleIds = [];
-  const tempRestoreFile = path.join(tempFolder, `restore_${tableName}.json`);
-
-  // create a backup file with 200 records
-  let fileContent = '';
-  for (let i = 0; i < limit; i += 1) {
-    const granule = fakeGranuleFactory();
-    fileContent += `${JSON.stringify(granule)}\n`;
-    granuleIds.push(granule.granuleId);
-  }
-  fs.writeFileSync(tempRestoreFile, fileContent);
+  await restore(tempBackupFile, tableName, 1);
```

Comment:
So this test is restoring to the same table that the data was initially populated in? Doesn't that mean it is possible that even if the restoration fails, the test would still pass from the initially populated data? Should the restore be to a new table since that also simulates the real world behavior?

Reply:
I updated that test and added a new one. Very good point.